### PR TITLE
#160344600 chore: add status to the responses

### PR DESCRIPTION
**Pull request by d-beloved**
#### What does this PR do?

- Add status to the existing responses in the app
- `status: success` when response is OK and `status: error` when a response is failing

### CODE REVIEW

**Review by emekafredy — COMMENTED** [review]
LGTM

File: server/controllers/article.js [hunk 1]
```diff
@@ -193,14 +197,14 @@ class ArticleController {
...
-            errors: { message: 'article not found' }
+            errors: { message: 'article not found', status: 'error' }
           });
         }
 
         // check if article belongs to current user
         if (parseInt(article.authorId, 10) !== parseInt(req.userId, 10)) {
           return res.status(403).json({
-            errors: { message: 'forbidden from editing another user\'s article' }
+            errors: { message: 'forbidden from editing another user\'s article', status: 'error' }
```
Comment (AdmiralOnu) [inline server/controllers/article.js]: "status" should be part of the response body, not a an attribute of error.

File: server/controllers/article.js [hunk 2]
```diff
@@ -193,14 +197,14 @@ class ArticleController {
         // return 404 if article not found
         if (!article) {
           return res.status(404).json({
-            errors: { message: 'article not found' }
+            errors: { message: 'article not found', status: 'error' }
```
Comment (AdmiralOnu) [inline server/controllers/article.js]: "status" should be part of the json response body, not an attribute of errors

File: server/controllers/article.js [hunk 3]
```diff
@@ -234,23 +238,25 @@ class ArticleController {
...
             message: 'article not found',
+            status: 'error'
           });
         }
 
         // check if article belongs to current user
         if (parseInt(article.authorId, 10) !== parseInt(req.userId, 10)) {
           return res.status(403).json({
-            errors: { message: 'forbidden from deleting another user\'s article' }
+            errors: { message: 'forbidden from deleting another user\'s article', status: 'error' }
```
Comment (AdmiralOnu) [inline server/controllers/article.js]: same as above

**Review by gonuoha — CHANGES_REQUESTED** [review]
error/errors attribute of response body should be consistent

**Review by gonuoha — APPROVED** [review]

**Pull request merged**